src/components/Timeline/src/styles/step-content.css

```diff
@@ -0,0 +1,8 @@
+.denhaag-timeline__step-content:not(:empty) {
+  margin-block-start: var(--denhaag-timeline-step-content-margin);
+  color: var(--denhaag-timeline-step-content-color);
+}
+
+.denhaag-timeline__step-content:empty {
+  margin-block-start: 0;
```

Comment:
This styling looks like it should belong in the MUI overrides of this class.

Reply:
moved